src/form/wrm.jl

```diff
@@ -256,9 +254,11 @@ function constraint_voltage(pm::GenericPowerModel{SDPDecompForm}, nw::Int, cnd::
             wr_jj = WR[2, 2]
             wr_ij = WR[1, 2]
             wi_ij = WI[1, 2]
+            wi_ji = WI[2, 1]
 
             # standard SOC form (Mosek doesn't like rotated form)
-            @constraint(pm.model, (wr_ii + wr_jj)/2 >= norm([(wr_ii - wr_jj)/2; wr_ij; wi_ij]))
+            @constraint(pm.model, (wr_ii + wr_jj) >= norm([(wr_ii - wr_jj); 2*wr_ij; 2*wi_ij]))
+            @constraint(pm.model, wi_ij == -wi_ji)
```

Comment:
FYI, in my past experience adding this symmetry constraint was causing some numerical issues in the solvers.